### PR TITLE
Prettify README

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+name: CI
```
Comment (SirVer) [inline .github/workflows/main.yml:1]: Can you explain what this does?

File: .replit [hunk 2]
```diff
@@ -0,0 +1,2 @@
+language = "python3"
```
Comment (SirVer) [inline .replit:1]: I have no idea what his file does.

File: README.md [hunk 3]
```diff
@@ -1,19 +1,38 @@
 ![Build Status](https://github.com/SirVer/ultisnips/actions/workflows/main.yml/badge.svg)
```
Comment (SirVer) [inline README.md:1]: These are very opionated changes. Can you outline your thought process around each one?

Comment (EmilySeville7cfg) [reply README.md:1]: I wanted to prettify README and structure it better. My goal was to follow this template:

- badges: CI status, official chat, doc reference
- short project description and maybe some important notes (for instance where to get snippets in this case)
- main features
- integrations
- usage example
- installation (via Vim package system or custom package manager)
- defaults
- tutorials
- history notes (for those users who are really interested in them)

Comment (SirVer) [reply README.md:1]: Thanks for your suggestion, however I do not agree with this structure being better than the current README. I really appreciate you taking the time!

**Pull request closed without merging**